tests/ltp/Makefile

```diff
@@ -66,6 +66,24 @@ build:
 	$(MAKE) group-file
 	$(MAKE) add_host_users_and_groups
 
+all-ubuntu: native-build
+	$(MYST) mkext2 --force native_appdir native_ext2fs
+
+native-build: native-clone
+	cp build.mak native_appdir/Makefile
+	$(MAKE) -C native_appdir native-all
+	$(MAKE) password-file
+	$(MAKE) group-file
+	$(MAKE) add_host_users_and_groups
+
+native-build-one: native-clone
+	cp -p build.mak native_appdir/Makefile
+	$(MAKE) -C native_appdir native-one TEST=$(TEST)
+	sudo chown -R $(USER).$(USER) appdir
```

Comment:
Why is this not required for the native-build case but required for the native-build-one case?

Reply:
Hi Radhika, thank you for noticing this, removed some unnecessary commands in my revision